history/taskana-simplehistory-provider/src/test/java/acceptance/events/task/CreateHistoryEventOnTaskTransferAccTest.java

```diff
@@ -70,40 +82,48 @@ Stream<DynamicTest> should_CreateTransferredHistoryEvent_When_TaskIsTransferred(
           assertThat(events).hasSize(1);
           assertTransferHistoryEvent(
               events.get(0).getId(),
-              "WBI:100000000000000000000000000000000001",
+              q.getThird(),
```

Comment:
assign to variable (like done with second and fourth)

Reply:
Ok :)